apps/video/js/video.js

```diff
@@ -35,7 +43,6 @@ ids.forEach(function createElementRef(name) {
 
 dom.player.mozAudioChannelType = 'content';
 
-function $(id) { return document.getElementById(id); }
```

Comment:
I think it's probably better to leave this in this function if possible, instead of putting it into shared/. Why do we need to move it into the shared/ file?